docs/github_actions.md

```diff
@@ -15,5 +15,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
-      - uses: psf/black@stable
+      - uses: psf/black@stable # the default is equivalent to `black . --diff --check`.
+        with: # (optional - override the default params).
+          args: [".", "--diff", "--check"]
```

Comment:
I just copied and pasted this example config and it has a syntactical error: https://github.com/ichard26/testblackaction/actions/runs/312826169

Appears that `with.args` doesn't support arrays containing strings :/ 
> A string that defines the inputs for a Docker container. GitHub passes the args to the container's ENTRYPOINT when the container starts up. An array of strings is not supported by this parameter.

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswithargs

Reply:
Sorry about that, updated.